recipes/cyanure/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set version = "0.22.4" %}
+{% set tag = "v0.0.4" %}
+
+package:
+  name: "cyanure"
+  version: {{ version }}
+
+source:
+  url: https://github.com/thomasryck/cyanure/archive/{{ tag }}/cyanure-{{ version }}.tar.gz
+  sha256: f926e613873221bcc96438f3a50379d8ec30616e79ac5c15476edc6bf45b7b2e
+    
+build:
+  number: 0
+  script:
+    - "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - llvm-openmp  # [osx]
+    - python
+    - libgomp  # [linux]
+    - intel-openmp  # [win]
+  host:
+    - python
+    - libblas
+    - libcblas
+    - numpy >=1.18.0
+    - scipy
+    - scikit-learn
+    - pip
+  run:
```

Comment:
Please list all runtime requirements under `run`. I assume this includes `numpy`, `scipy` and `scikit-learn`. Also it looks like you are building against `numpy`, so please take a look at:
https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

Reply:
I think I fixed the dependencies according to the documentation.